website/common/script/content/loginIncentives.js

```diff
@@ -1,5 +1,7 @@
 import _ from 'lodash';
 
+const MAX_INCENTIVES = 101;
```

Comment:
cron.js has another const with the same name, but set as 100 instead. Should this be pulled out into a more globally-available constant?